sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntity.cs

```diff
@@ -14,13 +14,13 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public readonly struct LinkedEntity
     {
-        internal LinkedEntity(string name, string id, string language, string dataSource, Uri uri, IEnumerable<LinkedEntityMatch> matches)
+        internal LinkedEntity(string name, string id, string language, string dataSource, Uri url, IEnumerable<LinkedEntityMatch> matches)
```

Comment:
Should this be `Url url`?

Reply:
I wanted to rename the property but not change the type itself. In .NET we usually use type `Uri`